.env.sample

```diff
@@ -1,6 +1,6 @@
 # warning: dotenv crate doesn't override
 
-# uncomment with :8332 to use your own mempool
+# uncomment 2 lines below below to use the docker regtest node
 # CORE_URL=127.0.0.1:18332
```

Comment:
Typo?  Port `8332` (instead of `18332`)

Reply:
not a typo, from now we default to using the users mainnet. only when you start to develop you need to setup docker and the micro-services and then the sample app show you how to use the dockerized regest node instead.